CARL/source/Descriptor.h

```diff
@@ -353,34 +359,34 @@ namespace carl::descriptor
             return{};
         }
 
-        static double Distance(
+        static NumberT Distance(
             const EgocentricWristOrientation& a,
             const EgocentricWristOrientation& b,
-            gsl::span<const double> tuning)
+            gsl::span<const NumberT> tuning)
         {
-            return tuning[0] *
-                std::pow(a.m_egocentricTemporalOrientation.angularDistance(b.m_egocentricTemporalOrientation), 2.);
+            auto distance = a.m_egocentricTemporalOrientation.angularDistance(b.m_egocentricTemporalOrientation);
+            return calculateDistance(distance, tuning[0]);
         }
 
         EgocentricWristOrientation() = default;
 
     private:
+        static inline constexpr auto calculateDistance{ createDistanceFunction(0.17453, 0.5236) };
         trivial::Quaternion m_egocentricTemporalOrientation{};
 
-        EgocentricWristOrientation(const InputSample& sample, const InputSample& priorSample)
+        EgocentricWristOrientation(const InputSample& sample, const InputSample&)
         {
             constexpr bool isLeftHanded = Handedness == Handedness::LeftHanded;
             auto wristPose = getWristPose<Handedness>(sample);
-            auto priorWristPosition =
-                (isLeftHanded ? priorSample.LeftWristPose.value() : priorSample.RightWristPose.value()).translation();
-            auto& priorHmdPose = priorSample.HmdPose.value();
+            auto wristPosition = wristPose.translation();
+            auto& hmdPose = sample.HmdPose.value();
 
-            auto ets = EgocentricTemporalSpace::getPose(priorWristPosition, priorHmdPose);
+            auto ets = EgocentricTemporalSpace::getPose(wristPosition, hmdPose);
```

Comment:
This change switches the egocentric temporal space from being that of the prior frame to that of the current frame. This prevents this descriptor from being framerate-dependent.